frontend/src/pages/Login.tsx

```diff
@@ -44,30 +44,31 @@ function Login() {
     const loginReq: LoginRequest = { credentials };
 
     axios
-      .post<LoginResponse>("/user/login", loginReq)
+      .post<LoginResponse>("/user/login", loginReq, {
+        withCredentials: true
+      })
       .then((res) => {
         const { errorCode, errorMessage } = res.data;
         if (errorCode) {
           throw new Error(errorMessage);
         }
 
-        const { user, sessionToken } = res.data;
+        const { user } = res.data;
 
         if (!user) {
           throw new Error("Something went wrong.");
         }
 
-        // Set cookie to axios instance
         const now = new Date();
-        setCookies("session_token", sessionToken, {
+        setCookies("session_token", "loggedin", {
           path: "/",
           expires: new Date(now.setDate(now.getTime() + 1000 * 86400)),
           domain: "127.0.0.1",
           secure: false,
         });
 
         // Store user information on redux
-        dispatch(login({ sessionToken, user }));
+        dispatch(login({ user, sessionToken: "loggedin" }));
```

Comment:
If `sessionToken` has no significance, maybe we can take it out of the redux slice? (I can handle this change if required)

Then to check if a user is logged in, we will check for the [existence of the user object](https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g33/blob/e1ab1b42322875fa44504b16b2f78d83c337d063/frontend/src/feature/user/userSlice.ts#L7)

Reply:
We'll change it in the next sprint.